packages/kuma-gui/src/app/data-planes/data/DataplaneNetworking.ts

```diff
@@ -94,7 +93,7 @@ export const DataplaneNetworking = {
             // inbound address, advertisedAddress, networkingAddress because externally accessible address
             addressPort: `${item.address ?? networking.advertisedAddress ?? networking.address}:${item.port}`,
             // inbound serviceAddress, inbound address, networkingAddress because the internal services accessible address
-            serviceAddressPort: `${item.serviceAddress ?? address}:${port}`,
+            serviceAddressPort: `${item.serviceAddress ?? address}:${item.servicePort ?? item.port}`,
```

Comment:
I don't think we should ever use the `servicePort` for when displaying/rendering/calculating inbounds